app/views/projects/show.html.erb

```diff
@@ -45,39 +34,85 @@
 <% end %>
 
 <dl class="dl-horizontal">
-  <dt><strong><%= t('.research_assistants', default:'Research Assistants') %>:</strong> </dt>
+  <dt><%= 'Actions' %>:</dt>
+  <br>
   <dd>
-    <% @project.users.sort { |a,b| a.name <=> b.name }.each do |user| %>
-        <div <%= "class='current'" if user == @user%>>
-          <% if user.is_wimi?%>
-              <div style="margin-bottom: 15px">
-                <%= user.name %>
-              </div>
-          <% end %>
-        </div>
+    <% content_for :modals do %>
+        <%= render 'invite_user' %>
+    <% end %>
+    <% if can?(:invite_user, @project) %>
+      <%= link_to t('project.invite_someone_to_project'), "", id: "open-popup", data: { toggle: "modal", target: "#invite-colleagues-modal" }, class: "btn-default link" %>
     <% end %>
   </dd>
+  <dd>
+    <% if can? :edit, @project %>
+        <%= link_to t('projects.form.show_all_working_hours', default:'Show all working hours'),
+                    work_days_path(month: 1, year: 2016, project: @project), class: 'btn-default link' %>
+    <% end %>
+  </dd>
+  <dd>
+    <% if @project.users.include? current_user%>
+        <%= link_to t('.leave_project', default:'Leave Project'),
+                    sign_user_out_project_path(user_id: current_user.id, id: @project), method: :delete, class: 'btn-default link' %>
+    <% end %>
+  </dd>
+
 </dl>
 
 <dl class="dl-horizontal">
-  <dt><strong><%= t('.student_assistants', default:'Student Assistants') %>:</strong> </dt>
+  <dt><strong><%= t('.project_team', default:'Project Team') %>:</strong> </dt>
   <dd>
-    <% @project.users.sort { |a,b| a.name <=> b.name }.each do |user| %>
-        <div <%= "class='current'" if user == @user %>>
-          <% if user.is_hiwi?%>
-              <div style="margin-bottom: 15px">
-                <%= user.name %>
-                <% if can? :edit, @project or user == current_user%>
-                  <%= link_to t('projects.form.show_working_hours', default:'Show working hours'), work_days_path(month: 1, year: 2016, project: @project.id), class: 'btn btn-warning btn-xs' %>
-                <% end %>
-              </div>
-          <% end %>
-        </div>
-    <% end %>
+    <div class="form-group">
+      <table class='table table-striped' width="1000px">
+        <thead>
+        <tr>
+          <th width="600px"><%= t('projects.form.wimi', default: 'Research Assistants:') %></th>
+        </tr>
+        </thead>
+        <tbody>
+        <% @project.users.order('last_name').each do |user| %>
+            <% if user.is_wimi?%>
+                <tr>
+                  <td>
+                    <%= link_to user.name, user  %>
```

Comment:
In den Mockups kann man Wimis zumindest austragen.
Aber einen Austragen-Link gibt es auch bei HiWis nicht.